main.go

```diff
@@ -34,6 +35,8 @@ var (
 	flagEmailFrom      string
 	flagEmailFromName  string = "WireGuard UI"
 	flagSessionSecret  string
+	flagClientMTU      int
+	flagAllowedIPs     string
```

Comment:
Can you also store these values in the database? They should be configurable not only from CLI but also UI.